.buildkite/create_integration_testing_worksheet.py

```diff
@@ -0,0 +1,48 @@
+"""
+# Requirements:
+    * Generate a service account key for your Google API credentials, then create environment variable GOOGLE_APPLICATION_CREDENTIALS.
+        - e.g export GOOGLE_APPLICATION_CREDENTIALS=/path/to/credentials.json.
+
+# Environment Variable/s:
+    * GOOGLE_APPLICATION_CREDENTIALS = Your service account key.
+"""
+
+from oauth2client.service_account import ServiceAccountCredentials
+import gspread
+import os
+
+
+WKS_CREDENTIALS = os.getenv("GOOGLE_APPLICATION_CREDENTIALS")
+WKS_TAG = os.getenv("BUILDKITE_TAG")
+
+WKS_TEMPLATE_TAG = 'release-v0.11.x'
+WKS_NAME = "Integration testing with Gherkin scenarios"
+WKS_INDEX_COPY = 0
+WKS_CELLS_RANGE = 100
+WKS_COLUMN = 'B'
```

Comment:
`INDEX_COPY`, `CELLS_RANGE`, and `COLUMN` could all use more descriptive variable names. For example, `SHEET_TEMPLATE_INDEX`, `MAX_ROW_COUNT`, and `FEATURE_LINKS_COLUMN`.